lib/awful/widget/prompt.lua

```diff
@@ -28,6 +29,9 @@ local function run(promptbox)
         exe_callback        = function (...)
             promptbox:spawn_and_handle_error(...)
         end,
+        fg_cursor           = beautiful.prompt_fg_cursor,
+        bg_cursor           = beautiful.prompt_bg_cursor,
+        font                = beautiful.prompt_font,
```

Comment:
I am fine with having fg/bg here, but those 3 really belong to `awful.prompt`. There is already a "fallback chain" over there. This make no sense to have them here.